day2/question2/solution2.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+var wg sync.WaitGroup
+func add(sum *int, rating int){
+	*sum+=rating
+	wg.Done()
+}
+
+func main(){
+	ratings := []int{2,5,3,4,2,3,2,3,4,5,5,5,3,1,4}
```

Comment:
Suggestion: Add a for loop instead of single array declaration

Reply:
Yeah, did that